src/test/resources/testDockerignore/.dockerignore

```diff
@@ -1 +1 @@
-b
\ No newline at end of file
+*/b
```

Comment:
According to `.dockerignore` doc:
> the root of the context is considered to be both the working and the root directory

then pattern `b` excludes all files and directories that are named b in root directory of build context.
However, in [this test](https://github.com/docker-java/docker-java/blob/master/src/test/java/com/github/dockerjava/core/command/BuildImageCmdImplTest.java#L191) the file named b is under directory a, so it should not be ignored. After my changes this tests fails.

Pattern `*/b`  works as expected (and described in the [doc](http://docs.docker.com/engine/reference/builder/#dockerignore-file) see `*/temp*` example)